snmp/datadog_checks/snmp/data/profiles/_generic-host-resources.yaml

```diff
@@ -65,3 +65,7 @@ metrics:
         column:
           OID: 1.3.6.1.2.1.25.3.3.1.1
           name: hrProcessorFrwID
+      - tag: deviceindex
+        column:
+          OID: 1.3.6.1.2.1.25.3.2.1.1
+          name: hrDeviceIndex
```

Comment:
Looks like hrDeviceIndex and hrProcessorFrwID/hrProcessorLoad are from different tables.

https://www.circitor.fr/Mibs/Mib/H/HOST-RESOURCES-MIB.mib

But I think your change should work because hrDeviceEntry and hrProcessorEntry they have the exact same index:

```
hrDeviceEntry OBJECT-TYPE
    SYNTAX     HrDeviceEntry
    MAX-ACCESS not-accessible
    STATUS     current
    DESCRIPTION
        "A (conceptual) entry for one device contained by the
        host.  As an example, an instance of the hrDeviceType
        object might be named hrDeviceType.3"
    INDEX { hrDeviceIndex }
    ::= { hrDeviceTable 1 }

hrProcessorEntry OBJECT-TYPE
    SYNTAX     HrProcessorEntry
    MAX-ACCESS not-accessible
    STATUS     current
    DESCRIPTION
        "A (conceptual) entry for one processor contained by
        the host.  The hrDeviceIndex in the index represents
        the entry in the hrDeviceTable that corresponds to the
        hrProcessorEntry.

        As an example of how objects in this table are named,
        an instance of the hrProcessorFrwID object might be
        named hrProcessorFrwID.3"
    INDEX { hrDeviceIndex }
    ::= { hrProcessorTable 1 }
```